src/services/audius-api-client/AudiusAPIClient.ts

```diff
@@ -19,6 +27,48 @@ type GetTrendingArgs = {
   genre?: string
 }
 
+type GetFollowingArgs = {
+  profileUserId: string
+  currentUserId?: string
```

Comment:
I forgot this on trending, but I have an 'OpaqueID' type that we could use for this